testsuite/integration/manualmode/pom.xml

```diff
@@ -126,6 +131,12 @@
             <artifactId>jboss-jaxws-api_2.3_spec</artifactId>
             <scope>test</scope>
         </dependency>
+<!--        <dependency>
```

Comment:
I'd prefer to remove this instead of commenting out.

Reply:
ok